website/docs/Installation.md

```diff
@@ -159,11 +161,27 @@ pip install pyautogen
 
 `pyautogen<0.2` requires `openai<1`. Starting from pyautogen v0.2, `openai>=1` is required.
 
-<!--
-or conda:
+### Code execution with Docker (default)
+
+Even if you install AutoGen locally, we highly recommend using Docker for [code execution](FAQ.md#code-execution).
+
+The default behaviour for code-execution agents is for code execution to be performed in a docker container. If you want to run the code locally (not recommended) then `use_docker` can be set to `False` in `code_execution_config` for each code-execution agent, or set `AUTOGEN_USE_DOCKER` to `False` as an environment variable.
```

Comment:
We need to add this comment to the starting examples referred to in REAMDE and Getting-Started. They didn't set `use_docker` and will result in failure for first-time users.

Reply:
updated with use docker turned off just for a better user experience to begin with